chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -126,6 +127,7 @@
 <!ENTITY zotero.item.textTransform.titlecase			"Title Case">
 <!ENTITY zotero.item.textTransform.sentencecase			"Sentence case">
 <!ENTITY zotero.item.creatorTransform.nameSwap      	"Swap First/Last Names">
+<!ENTITY zotero.item.openURL							"Open in Browser">
```

Comment:
And actually, the tooltip for the DOI and URL labels is already "Go to this item online", so "View Online" is consistent with that.